src/components/shared/inputs/ImageUploader.tsx

```diff
@@ -53,7 +53,7 @@ export default function ImageUploader({
                 borderRadius: theme.radii.md,
               }}
               src={url}
-              alt="Uploaded image"
+              alt="Uploaded user content"
```

Comment:
This wasn't in the initial warning, but it was showing up. It might have just been a stale yarn cache that didn't catch this rule the first time because my bundle/lint changed so dramatically between branches.

Reply:
hah, nice. I like this rule :D